plugins/altstatusbar.koplugin/main.lua

```diff
@@ -0,0 +1,271 @@
+local T = require("ffi/util").template
+local logger = require("logger")
+local _ = require("gettext")
+
+local AltStatusBar = WidgetContainer:new{
+    name = "altstatusbar",
+    is_doc_only = true,
+}
+
+function AltStatusBar:isEnabled()
+    return G_reader_settings:readSetting("copt_status_line") == 0
+end
+
+function AltStatusBar:onReadSettings()
+    if self.document.provider == "crengine" then
+        self.enabled = self:isEnabled()
+        self.title = G_reader_settings:readSetting("cre_header_title") or 1
+        self.clock = G_reader_settings:readSetting("cre_header_clock") or 1
+        self.page_number = G_reader_settings:readSetting("cre_header_page_number") or 1
+        self.page_count = G_reader_settings:readSetting("cre_header_page_count") or 1
+        self.reading_percent = G_reader_settings:readSetting("cre_header_reading_percent") or 0
+        self.battery = G_reader_settings:readSetting("cre_header_battery") or 1
+        self.battery_percent = G_reader_settings:readSetting("cre_header_battery_percent") or 0
+        self.chapter_marks = G_reader_settings:readSetting("cre_header_chapter_marks") or 1
+
+        self.ui.document._document:setIntProperty("window.status.title", self.title)
+        self.ui.document._document:setIntProperty("window.status.clock", self.clock)
+        self.ui.document._document:setIntProperty("window.status.pos.page.number", self.page_number)
+        self.ui.document._document:setIntProperty("window.status.pos.page.count", self.page_count)
+        self.ui.document._document:setIntProperty("crengine.page.header.chapter.marks", self.chapter_marks)
+        self.ui.document._document:setIntProperty("window.status.battery", self.battery)
+        self.ui.document._document:setIntProperty("window.status.battery.percent", self.battery_percent)
+        self.ui.document._document:setIntProperty("window.status.pos.percent", self.reading_percent)
+
+        UIManager:broadcastEvent(Event:new("SetStatusLine", self.document.configurable.status_line, false))
+
+        self.ui.menu:registerToMainMenu(self)
+    else
+        logger.dbg("AltStatusBar disabled")
+    end
+end
+
+function AltStatusBar:addToMainMenu(menu_items)
+    menu_items.alt_status_bar = {
+        sorting_hint = "setting",
+        text = _("Alt Status Bar"),
+        checked_func = function()
+            return self:isEnabled()
+        end,
+        sub_item_table = {
+            {
+                text = _("Enable top status bar"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self:isEnabled()
+                end,
+                callback = function()
+                    local old_status_line = G_reader_settings:readSetting("copt_status_line") or 1
+                    G_reader_settings:saveSetting("copt_status_line", 1 - old_status_line)
+                end,
+            },
+            {
+                text = _("Exclude top status bar for this document"),
```

Comment:
So, I guess this is duplicating the bottom menu toggle? As a way to have all this setable from the same place?

Reply:
Yes, but the `Exclude top status bar for this document` could be omitted, then I go for activatint the AltStatusBar for the document together with the first entry. -> Then all entries would be on one page :)